libcodechecker/server/client_db_access_handler.py

```diff
@@ -78,11 +161,14 @@ def construct_report_filter(report_filters):
             AND.append(Report.severity == report_filter.severity)
         if report_filter.bugHash:
             AND.append(Report.bug_id == report_filter.bugHash)
+        if report_filter.status is not None:
+            # status value can be 0
```

Comment:
This warning is here to watch out for the if statement because the `report_filter.status` can be `0` which is a valid enum value, but it would be interpreted as `False` by the if statement.